charts/prometheus-postgres-exporter/templates/_helpers.tpl

```diff
@@ -61,3 +61,12 @@ Return the appropriate apiVersion for rbac.
 {{- print "rbac.authorization.k8s.io/v1beta1" -}}
 {{- end -}}
 {{- end -}}
+
+{{/* Get Policy API Version */}}
+{{- define "prometheus-postgres-exporter.pdb.apiVersion" -}}
+{{- if .Capabilities.APIVersions.Has "policy/v1" }}
+    {{- print "policy/v1" -}}
```

Comment:
I'd suggest to indent in the same way on the two lines or not to indent at all. Thank you!